README.md

```diff
@@ -31,19 +37,27 @@ In the conda environment defined in the root\environment.yml, run the following:
 
 `python src\main.py --instructions full\path\to\instruction.json`
 
-## Running tests
-In the conda environment defined in the root\environment.yml, run the following in the repository root folder:
+## Tests
+Tests exist for stand alone functionality (i.e. fetch lidar), and complete processing chain (i.e. creating a DEM from LiDAR files within a shapefile). A 'benchmark_dem.nc' is uploaded for each test when a DEM is generated. This is stored using git LTS as these file are not human readable. 
+
+### Automated testing
+[Github Actions](https://docs.github.com/en/actions) are used to run tests after each push to remote (i.e. github). [Miniconda](https://github.com/marketplace/actions/setup-miniconda) from the GitHub Actions marketplace is used to install the package dependencies. Linting with [Flake8](https://github.com/py-actions/flake8) and testing with [PyTest](https://docs.pytest.org/en/6.2.x/contents.html) is then performed. 
+
+Check the actions tab after you push to check if your tests run successfully.
```

Comment:
I like this documentation. Depending on the end goal at some point you may have to look at something bigger than READMEs, perhaps a wiki or some site dedicated to documentation. I don't think this is needed yet though

Reply:
Great suggestion. Yes - it's definitely getting a bit long! Probably a good documentation issue.